hadoop-client-modules/hadoop-client-integration-tests/pom.xml

```diff
@@ -52,6 +52,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.lz4</groupId>
+      <artifactId>lz4-java</artifactId>
+      <scope>test</scope>
```

Comment:
we can remove this now since we have the dependency in `hadoop-client-api`, is that right?